Core/include/Acts/Detector/CylindricalContainerBuilder.hpp

```diff
@@ -70,6 +76,20 @@ class CylindricalContainerBuilder : public IDetectorComponentBuilder {
   /// @return an outgoing detector component
   DetectorComponent construct(const GeometryContext& gctx) const final;
 
+  /// Create a new builder instance from a blueprint
+  ///
+  /// @param bpNode is the entry blue print node
+  /// @param logLevel is the logging output level for the builder tools
+  ///
+  /// @note no checking is being done on consistency of the blueprint,
+  /// it is assumed it has passed first through gap filling via the
+  /// blueprint helper.
+  ///
+  /// @return a cylindrical container builder representing this blueprint
+  static std::shared_ptr<const IDetectorComponentBuilder> createFromBlueprint(
```

Comment:
Technically, this could be a simple constructor I guess?

Reply:
Absolutely right, I though probably as part of a chain one could want a function, but I can change that to a constructor now.

Reply:
Changed to a constructor.